src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -167,36 +169,99 @@ function DeploymentConfigForm({ respondOnSuccess }) {
+                                        ...base,
+                                        color: 'var(--N900)',
+                                        backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
+                                    })
+                                },
+                                container: (base, state) => {
+                                    return ({
+                                        ...base,
+                                        width: '100%'
+                                    })
+                                },
+                            }}
+                            onChange={(selected) => selectChart(selected as { id: number, version: string, name: string })}
+                        />
+                    </>
+                    ):(
+                        <input autoComplete="off" value={selectedChart?.name} className="form__input" disabled />
+                    )}
+                    </div>
+                    <div className="flex left column">
+                        <div className="form__label">Chart version</div>
+                        <ReactSelect options={filteredChartVersions}
+                            isMulti={false}
+                            getOptionLabel={option => `${option.version}`}
+                            getOptionValue={option => `${option.id}`}
+                            value={selectedChart}
+                            components={{
+                                IndicatorSeparator: null
+                            }}
+                            styles={{
+                                control: (base, state) => ({
+                                    ...base,
+                                    boxShadow: 'none',
+                                    border: `solid 1px var(--B500)`,
+                                    width: '100%'
+                                }),
+                                option: (base, state) => {
+                                    return ({
+                                        ...base,
+                                        color: 'var(--N900)',
+                                        backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
+                                    })
+                                },
+                                container: (base, state) => {
+                                    return ({
+                                        ...base,
+                                        width: '100%'
+                                    })
+                                },
+                            }}
+                            onChange={(selected) => selectChart(selected as { id: number, version: string, name: string })}
+                        />
+                    </div>
+                </div>
+                <div className="deploymentConfig__warning flex fs-12 left pt-4 m-b-16">
+                    { isUnSet ? (
```

Comment:
Can be simplified to :

              { isUnSet && 
                        <>
                        <Warn className="icon-dim-16 mr-4 " />
                        Chart type cannot be changed once saved.
                        </>
                    }

No ternary operation is needed

Reply:
done